pkgs/development/python-modules/csv2md/default.nix

```
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "csv2md";
+  version = "1.1.1";
```

```suggestion
  version = "1.1.1";
  pyproject = true;
```
Could you please set `pyproject = true`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters